printing/printer.mli

```diff
@@ -104,6 +104,9 @@ val pr_pconstructor        : env -> pconstructor -> std_ppcmds
 
 
 (** Contexts *)
+(** Display compact contexts of goals (simple hyps on the same line) *)
+val compact_context : bool ref
+val is_compact_context : unit -> bool
```

Comment:
@Matafou well I think that the current style is mostly instead going on the other direction (exposing the ref and just using normal update operations), but I guess it doesn't really matter.